<?xml version="1.0"?>

<launch>

<arg name="vocabulary_path" default="$(find ORB_SLAM2v2)/Vocabulary/ORBvoc.txt"/>
<arg name="settings_path" default="$(find ORB_SLAM2v2)/config/Asus.yaml"/>
<arg name="localization"  default="false"/>

<node name="server_node0" pkg="ORB_SLAM2v2" type="server_node" args="$(arg vocabulary_path) $(arg settings_path)" output="screen">
	<param name="mapBinaryPath"	type="string"	value="$(find ORB_SLAM2v2)/server_orbmap.bin"/>
	<param name="mapOctomapPath"	type="string"	value="$(find ORB_SLAM2v2)/server_global_octomap.bt"/>
	<param name="mapPCLPath"	type="string"	value="$(find ORB_SLAM2v2)/optimized_pointcloud.pcd"/>
	<param name="ClientId"	type="int"	value="0"/>
</node>

</launch>
